js/foundation/foundation.reveal.js

```diff
@@ -68,7 +68,7 @@
         });
 
       S(document)
-        .on('click.fndtn.reveal', this.close_targets(), function (e) {
+        .on('click.fndtn.reveal keydown.fndtn.reveal', this.close_targets(), function (e) {
```

Comment:
Should really only be `ENTER` or `SPACE` as not all keys are expected to activate a button